docs/conf.py

```diff
@@ -0,0 +1,56 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'PGHoard'
+copyright = '2021, Aiven'
+author = 'Aiven'
+
+# The full version, including alpha/beta/rc tags
+release = '2.2.1'
```

Comment:
Would be nice to get that from the version file instead of needing to edit it twice.

Reply:
It looks like it is fetched from the git tag, so I used the version.py script at the root of the project.